src/test/java/com/wikia/webdriver/testcases/adstests/mobileadstests/TestSlotsMobile.java

```diff
@@ -76,4 +76,30 @@ public void TestLeaderboardAndInContentOnPage(
     ads.verifyImgAdLoadedInSlot(MOBILE_IN_CONTENT, medrecImgUrl);
     ads.verifyNoSlotPresent(MOBILE_PREFOOTER);
   }
+
+    @Test(
+            groups = {"MobileAds", "MercuryAds", "TestAdSlotsMobile_004", "TestAdSlotsMobile"},
+            dataProviderClass = MobileAdsDataProvider.class,
+            dataProvider = "mercuryConsecutivePageViews"
+    )
+    public void TestLeaderboardAndPrefooterOnConsecutivePageViews(
```

Comment:
I would go with different approach and I get rid of `MobileAdsBaseObject:mercuryNavigateToMainPage()` and `MobileAdsBaseObject:mercuryNavigateToAnArticle()`.
I'd create the 3 synthetic pages and navigate through them by clicking the link or just typing URLs to an address bar (depends on how we can reproduce the defect) and check that `MOBILE_TOP_LEADERBOARD ` and `MOBILE_PREFOOTER ` are still on the page.